examples/nextjs-hooks-with-typescript/README.md

```diff
@@ -0,0 +1,29 @@
+# Next.js Hooks with Typescript example
+
+## How to use
+
+Download the example [or clone the repo](https://github.com/mui-org/material-ui):
+
+```bash
+curl https://codeload.github.com/mui-org/material-ui/tar.gz/master | tar -xz --strip=2  material-ui-master/examples/nextjs-hooks-with-typescript
+cd nextjs-hooks-with-typescript
+```
+
+Install it and run:
+
+```bash
```

Comment:
Sometimes we are using `sh` over `bash`. I have seen React using sh.

![capture d ecran 2018-12-21 a 20 30 15](https://user-images.githubusercontent.com/3165635/50360122-42aa4d80-055f-11e9-9223-2472bc2c9388.png)

I would vote for replacing all the occurrences of bash with sh.